props/blaster.h

```diff
@@ -260,9 +233,65 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+   RefPtr<BufferedWavPlayer> auto_player_;
+
+  uint32_t auto_time_;
+  virtual void Fire() {
+    CheckEmpty();
+    if (empty_) return;
+#ifdef ENABLE_MOTION
+#ifdef BLASTER_JAM_PERCENTAGE
+    // If we're already jammed then we don't need to recheck. If we're not jammed then check if we just jammed.
+    is_jammed_ = is_jammed_ ? true : CheckJam(BLASTER_JAM_PERCENTAGE);
+    if (is_jammed_) {
+      SaberBase::DoEffect(EFFECT_JAM, 0);
+      return;
+    }
+#endif
+#endif
+    if (blaster_mode == MODE_AUTO) {
+      // Set up the autofire pairing if the font suits it.
+      SelectAutoFirePair(); 
+      SaberBase::SetLockup(LOCKUP_AUTOFIRE);
+      SaberBase::DoBeginLockup();
+      auto_firing_ = true;
+      auto_time_ = millis();
+
+      // GET auto.wav here
+      if (!auto_player_) {
```

Comment:
I think if auto_player_ is set here, we need to override it with the new one.

Reply:
so then just remove the if clause it seems.....